ocaml/xapi/xapi_ha_vm_failover.ml

```diff
@@ -15,27 +15,27 @@
 module D = Debug.Make(struct let name="xapi_ha_vm_failover" end)
 open D
 
-type task_result = (Rpc.t, exn) Result.result
+type task_result = (Rpc.t, exn) Stdlib.Result.t
```

Comment:
`Stdlib.Result.t` is just `result`: if you have a compiler new enough to have stdlib it will be new enough to have result.